src/main/cpp/commands/SetHickeyPos.cpp

```diff
@@ -0,0 +1,21 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#include "commands/SetHickeyPos.h"
+
+SetHickeyPos::SetHickeyPos(DooHickey* dooHickey, double position) : mDooHickey(dooHickey), mPosition(position) {
```

Comment:
Let's use a reference to the subsystem.

```suggestion
SetHickeyPos::SetHickeyPos(DooHickey& dooHickey, double position) : mDooHickey(dooHickey), mPosition(position) {
```